DANS-KNAW/easy-fedora-to-bag
# DD-947 + DD-948: log.csv improvements

Fixes EASY-947: log `OK` for exported non-strict bags
Fixes EASY-948: log `ignored` for datasets in the input but also on the skip-list

#### When applied it will...
* no longer have the option `fedora-versioned`
* changed `No payload files found` into 
  `Running with fedora-versioned and no payload found`
* 

#### Where should the reviewer @DANS-KNAW/easy start?

* Please compile with JDK-8 because of jibx:

      export JAVA_HOME=`/usr/libexec/java_home -v 1.8`

#### How should this be manually tested?

The following command from the root of the project. With all deasy datasets in `data/in.tx` except for `easy-dataset:10` which has a very large file, and some in the skip-list. Note that only `easy-dataset:9` has a second bag.

    run.sh -i data/in.txt --output-dir data/out --output-format SIP -l data/log.csv --skip-list data/skip.txt original-versioned

Repeat with `--strict`

#### Related pull requests on github

repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
